src/Knp/Rad/FixturesLoad/ResetDatabaseSchema.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Knp\Rad\FixturesLoad;
+
+use Doctrine\ORM\Tools\SchemaTool;
+use Doctrine\ORM\EntityManager;
+
+class ResetDatabaseSchema
```

Comment:
You should named it as a role, not as an action, do not create confusion between the class and the method.